src/findfile.c

```diff
@@ -48,8 +48,8 @@
  *
  * ATTENTION:
  * ==========
- *	Also we use an allocated search context here, this functions are NOT
- *	thread-safe!!!!!
+ *	Also we use an allocated search context here, these functions are NOT
+ *	thread-safe!
```

Comment:
I suspect this might have been Bram amusing himself:

“And all those exclamation marks, you notice? Five? A sure sign of someone who wears his underpants on his head.”
― Terry Pratchett, Maskerade